Adding info in Custom Order Status doc (#905)
magento/merchdocs: src/sales/order-status-custom.md

|Order Status|Order State||
|--- |--- |--- |
|Processing|processing|When the state of new orders is set to “Processing”, the _Automatically Invoice All Items_ option becomes available in the configuration.|

> |Suspected Fraud|fraud|Sometimes orders paid via PayPal, eway or any other payment gateway, mark the orders as Suspected Fraud. This means the order does not have invoice issued and the confirmation email is also not sent.|

Reviewer: eway is removed from 2.4 and deprecated in 2.3.

```suggestion
|Suspected Fraud|fraud|Sometimes orders paid via PayPal or another payment gateway are marked as Suspected Fraud. This means the order does not have invoice issued and the confirmation email is also not sent.|
```

Author: Done

> |Pending Payment|pending_payment|This is the status used if order is created and PayPal or similar payment method is used. This means that customer has been sent to payment gateway website but no return information has been received yet. This status will change when customer pays.|

Reviewer: ```suggestion
|Pending Payment|pending_payment|This is the status used if order is created and PayPal or similar payment method is used. This means that the customer was directed to the payment gateway website, but no return information has been received yet. This status will change when customer pays.|
```

Author: Done

|Payment Review|payment_review|This status appear when PayPal payment review is turned on.|
|Pending|pending|This status means no invoice and shipments have been submitted.|
|On Hold|holded|This status can only be assigned manually. You can put any order on hold.|

> |Complete|complete|This status means that order is crated, paid and shipped to customer.|

Reviewer: ```suggestion
|Complete|complete|This status means that order is crated, paid, and shipped to customer.|
```

Author: Done

|Closed|closed|This status indicates that an order have had a credit memo assigned to it and the customer has received a refund.|
|Canceled|canceled|This status is assigned manually by you or, on some payment gateways, when customer does not pay within the specified time.|

> |PayPal Canceled Reversal|payment_canceled_reversal|This status means that Paypal cancelled the reversal.|

Reviewer: ```suggestion
|PayPal Canceled Reversal|payment_canceled_reversal|This status means that PayPal canceled the reversal.|
```

Author: Done

|PayPal Reversed|paypal_reversed|Reversal status means that PayPal reversed the transaction.|

> |Pending PayPal|pending_paypal|This status means that order has been received by PayPal, but payment has not yet been processed.|

Reviewer 2: src/images/images/config-services-oauth-authentication-locks.png
